src/commands/logs/build.mts

```diff
@@ -0,0 +1,92 @@
+import { OptionValues } from 'commander'
+import inquirer from 'inquirer'
+
+import { log, chalk } from '../../utils/command-helpers.mjs'
+import { getWebSocket } from '../../utils/websockets/index.mjs'
+import BaseCommand from '../base-command.mjs'
+
+export function getName({ deploy, userId }: { deploy: any; userId: string }) {
+  let normalisedName = ''
+  const isUserDeploy = deploy.user_id === userId
+
+  switch (deploy.context) {
```

Comment:
Nit: It'd be great to have the potential values be an enum or something so that it's easier to keep track of all the potential values this could be as well as ensure that we factor in all possible cases in future code that looks similar to this

Reply:
Yeah a string literal may be appropriate here but I think it's a bit premature atm, when we expand out be worth typing properly